docs/guides/cloud/data-storage-and-masking.mdx

```diff
@@ -0,0 +1,169 @@
+---
+title: Data Storage and Controls
+sidebar_position: 60
+sidebar_label: Data Storage and Controls
+---
+
+:::info
+
+## <Icon name="graduation-cap" /> What you'll learn
+
+- What data does Cypress Cloud store?
```

Comment:
Tiny thing, but I don't believe we frame these points as questions in any other page.